FileAPI/url/url-with-fetch.any.js

```diff
@@ -1,26 +1,22 @@
 // META: script=resources/fetch-tests.js
 
-async function garbageCollect() {
-  if (typeof TestUtils !== 'undefined' && TestUtils.gc) {
-    await TestUtils.gc();
-  } else if (self.gc) {
-    await self.gc();
-  } else if (self.GCController) {
-    // Present in some WebKit development environments
-    await GCController.collect();
-  } else {
-    var gcRec = function (n) {
-      if (n < 1)
-        return {};
-      var temp = {i: "ab" + i + (i / 100000)};
-      temp += "foo";
-      gcRec(n-1);
-    };
-    for (var i = 0; i < 1000; i++)
-      gcRec(10);
+function garbageCollect() {
```

Comment:
Maybe we should get a common helper function rather than having multiple duplicates?